style.css

```diff
@@ -1,245 +1,328 @@
-.logo {
-  color: #6071fe;
-  font-size: 1.5em;
-}
+@media (min-width: 768px) {
+  .logo {
+    color: #6071fe;
+    font-size: 1.5em;
+  }
 
-body {
-  background: #f7f7f7;
-  font-family: Poppins, sans-serif;
-}
+  body {
+    background: #f7f7f7;
+    margin: 0%;
+    font-family: Poppins, sans-serif;
+  }
 
-.separator {
-  width: 100%;
-}
+  .separator {
+    width: 100%;
+  }
 
-.navigation {
-  position: fixed;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  padding-left: 50px;
-  padding-right: 50px;
-  padding-top: 10px;
-  background: white;
-  top: 0;
-  left: 0;
-  right: 0;
-}
+  .navigation {
+    position: fixed;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-left: 50px;
+    padding-right: 50px;
+    padding-top: 10px;
+    background: white;
+    top: 0;
+    left: 0;
+    right: 0;
+  }
 
-.links,
-ul {
-  list-style: none;
-  margin: 0;
-  padding: 0.5em;
-  display: flex;
-}
+  .links,
+  ul {
+    list-style: none;
+    margin: 0;
+    padding: 0.5em;
+    display: flex;
+  }
 
-.links li {
-  margin-right: 20px;
-}
+  .links li {
+    margin-right: 20px;
+  }
 
-a {
-  color: #6071fe;
-  text-decoration: none;
-}
+  a {
+    color: #6071fe;
+    text-decoration: none;
+  }
 
-.hamburger {
-  display: none;
-}
+  .hamburger {
+    display: none;
+  }
 
-.header {
-  padding-left: 300px;
-  padding-right: 300px;
-  padding-top: 200px;
-  height: 100vh;
-  border-radius: 0 0 0 100px;
-  background: white;
-  background-size: contain;
-  background-image: url("images/header.png");
-}
+  .header {
+    padding-left: 300px;
+    padding-right: 300px;
+    padding-top: 200px;
+    height: 100vh;
+    border-radius: 0 0 0 100px;
+    background: white;
+    background-size: contain;
+    background-image: url("images/header.png");
+  }
 
-.headline {
-  font-weight: 600;
-  font-size: 4em;
-}
+  .headline {
+    font-weight: 600;
+    font-size: 4em;
+  }
 
-.headline-description {
-  font-weight: 100;
-  font-size: 2em;
-  padding-top: 10px;
-  padding-bottom: 10px;
-}
+  .headline-description {
+    font-weight: 100;
+    font-size: 2em;
+    padding-top: 10px;
+    padding-bottom: 10px;
+  }
 
-.connect {
-  font-weight: 500;
-  padding-top: 20px;
-  padding-bottom: 20px;
-  font-size: 1.2em;
-  color: #6071fe;
-  text-transform: uppercase;
-}
+  .connect {
+    font-weight: 500;
+    padding-top: 20px;
+    padding-bottom: 20px;
+    font-size: 1.2em;
+    color: #6071fe;
+    text-transform: uppercase;
+  }
 
-.social-links {
-  display: flex;
-  width: 20%;
-  justify-content: space-between;
-  align-items: center;
-  margin-top: 20px;
-  margin-bottom: 40px;
-}
+  .social-links {
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you adjust the padding and margin spaces of your social-links to make it similar to the Figma design.

| Your Website  | Figma Design |
| ------------- | ------------- |
| <img width="179" alt="image" src="https://user-images.githubusercontent.com/52934128/186464701-1a92a1a7-729c-41a6-8a36-6c619d02ee1e.png">  | <img width="308" alt="image" src="https://user-images.githubusercontent.com/52934128/186464593-83b4c658-2107-418b-bc1c-dbf70d82deee.png">  |